File: Exercism/python/roman-numerals/roman_numerals.py
ROMAN_NUMBERS = {
        'M': 1000,
        'CM': 900,
        'D': 500,
        'CD': 400,
        'C': 100,
        'XC': 90,
        'L': 50,
        'XL': 40,
        'X': 10,
        'IX': 9,
        'V': 5,
        'IV': 4,
        'I': 1
}


def roman(number: int) -> str:
    """Converts normal number to Roman Numerals."""
    result = ''
    for roman, arabic in ROMAN_NUMBERS.items():
        count, number = divmod(number, arabic)
        result += roman * count
    return result

File: Exercism/javascript/roman-numerals/roman-numerals.js
'use strict'

const ALPHA_TO_ROMAN = new Map()
  .set('M', 1000)
  .set('CM', 900)
  .set('D', 500)
  .set('CD', 400)
  .set('C', 100)
  .set('XC', 90)
  .set('L', 50)
  .set('XL', 40)
  .set('X', 10)
  .set('IX', 9)
  .set('V', 5)
  .set('IV', 4)
  .set('I', 1)

export const toRoman = (number) => {
  let result = ''
  for (const [roman, arabic] of ALPHA_TO_ROMAN.entries()) {
    result += roman.repeat(Math.floor(number / arabic))
    number %= arabic
  }
  return result
};
